python/StructureSolver.py

```diff
@@ -54,7 +54,7 @@
 t = 0
 
 print("Structure: init precice...")
-precice_tau = interface.initialize()
+precice_dt = interface.initialize()
```

Comment:
Maybe we should add a comment here: `# precice defines timestep size of solver via precice-config.xml`